src/main/java/com/stripe/model/Coupon.java

```diff
@@ -26,6 +27,9 @@
   @SerializedName("amount_off")
   Long amountOff;
 
+  @SerializedName("applies_to")
+  Restrictions appliesTo;
```

Comment:
Wonder why this is called Restrictions and not CouponAppliesTo ?

Reply:
or maybe just an inner class `AppliesTo`?

Reply:
OH NO ugh I must have named it wrong and that wasn't caught in the PR. I guess I have to fix it internally now. sadness.